index.html

```diff
@@ -127,7 +128,259 @@ <h2>Multi-Post Stories</h2>
                                 </ul>
                             </li>
                         </ul>
-                        <button type="button">See Project</button>
+                        <button type="button" class="button">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="myProjects">
```

Comment:
- [ ] I like the color you have in your background :+1: , however, is a project requirement to follow the template as much as possible. Please let's fix this.
## Figma background color #fff
![image](https://user-images.githubusercontent.com/28109626/148608117-1e1fe015-46c5-4013-b6b7-5a2aeb89b65a.png)

## Your project background color #e5e5e5
![image](https://user-images.githubusercontent.com/28109626/148607872-dea2fdc2-0584-4ee2-8e30-75d69e51ef5f.png)

Reply:
Hello @mike2611 Thank you for this review.
- I have changed my background color to #fff